fix for compilation error:
error: no instance of constructor "testing::AssertionResult::AssertionResult" matches the argument list
author: Kenneth Hoste (HPC-UGent)
--- test/lib/io/TestBam.cpp.orig	2017-04-03 18:06:43.355151223 +0200
+++ test/lib/io/TestBam.cpp	2017-04-03 18:06:09.154756888 +0200
@@ -107,7 +107,7 @@
     size_t size = in.tellg();
     in.seekg(0, std::ios::beg);
     std::vector<char> buf(size);
-    ASSERT_TRUE(in.read(buf.data(), size));
+    ASSERT_TRUE((bool)in.read(buf.data(), size));
     buf.push_back(0); // make sure buffer is null terminated
     EXPECT_STREQ(samData.c_str(), buf.data());
 
